src/api.ts

```diff
@@ -52,6 +52,32 @@ export type TransactionRequest = {
   ccipReadEnabled?: boolean
 }
 
+/**
+ * Timing options
+ */
+export interface TimingStrategyMinWaitTime {
+  strategy: 'minWaitTime'
+  minWaitTimeMs: number
+  startingExpectedResults: number
+  reduceEveryMs: number
+}
+
+export type TimingStrategyMinWaitTimeString =
+  `minWaitTime-${number}-${number}-${number}`
```

Comment:
Should we maybe stick to one format and use `TimingStrategyMinWaitTimeString` in both `routes` and `quote` requests? I like `TimingStrategyMinWaitTime` more, but we can't use it in the quote request, yeah...